app/instance-initializers/global.js

```diff
@@ -0,0 +1,10 @@
+export function initialize(appInstance) {
+    // Make application instance available globally to allow dynamic registration of new models.
+    //  Refs http://emberigniter.com/accessing-global-object-in-ember-cli-app/  ; TODO: Is there a better way?
+    window.App = appInstance;
```

Comment:
Noting this for self: it seems that the window object is indeed cleared when the document is unloaded (eg navigating to another page). This is good (future sites not having access to the global while the "window" remains open), but if we can do it without binding to window that's still preferred.

http://docstore.mik.ua/orelly/web/jscript/ch11_06.html

See also https://github.com/ember-cli/ember-export-application-global/blob/master/app/initializers/export-application-global.js